packages/typescript/src/internal/ts_project.bzl

```diff
@@ -0,0 +1,240 @@
+"ts_project rule"
+
+load("@build_bazel_rules_nodejs//:providers.bzl", "DeclarationInfo", "NpmPackageInfo", "run_node")
+
+_ATTRS = {
```

Comment:
I think we should add "args" in general to rules to allow users to pass their own arguments as
1) config files do not always support all arguments. I recently added "args" to rollup for this reason so 
Angular could pass `--silent` which is command line one.
2) users may want to override config file settings with command line arguments in some cases
General pattern is
```
    "args": attr.string_list(
        doc = """Command line arguments to pass to <tool name>. Can be used to override config file settings.
```
and 
```
```
    # See CLI documentation at https://rollupjs.org/guide/en/#command-line-reference
    args = ctx.actions.args()

    # Add user specified arguments *before* rule supplied arguments
    args.add_all(ctx.attr.args)
```
where `args` is passed to `run_node`

Reply:
added `args` pass-through with a delightful test coverage